dask/dataframe/core.py

```diff
@@ -86,6 +86,20 @@
 pd.set_option("compute.use_numexpr", False)
 
 
+def _numeric_only(func):
+    @wraps(func)
+    def wrapper(self, *args, **kwargs):
+        if kwargs.get("numeric_only") is False:
```

Comment:
The default value here is always `None`, which gives the previous behaviour of all the affected functions?

Reply:
yep!

Reply:
OK, then I'm totally on board with this PR, I just recommend some text here in the decorator function to describe what its for.